detector/hmac.go

```diff
@@ -2,25 +2,26 @@ package detector
 
 import (
 	"fmt"
-	"io/ioutil"
+	"os"
 	"path"
 	"strings"
 	"sync"
 	"time"
 
-	"github.com/maximbaz/yubikey-touch-detector/notifier"
+	"github.com/deckarep/golang-set"
 	"github.com/rjeczalik/notify"
-	"github.com/scylladb/go-set/strset"
 	log "github.com/sirupsen/logrus"
+
+	"github.com/maximbaz/yubikey-touch-detector/notifier"
 )
 
 // WatchHMAC watches when YubiKey is waiting for a touch on a HMAC request
 func WatchHMAC(notifiers *sync.Map) {
 	devicesEvents := initInotifyWatcher("HMAC", "/dev", notify.Create, notify.Remove)
 	defer notify.Stop(devicesEvents)
 
-	yubikeyHidrawDevices := strset.New()
-	if devices, err := ioutil.ReadDir("/dev"); err == nil {
+	yubikeyHidrawDevices := mapset.NewSet()
```

Comment:
It was painful to review a pull request that combines lots of unrelated style changes, library replacements, and actual new feature all in one big patch. It would be very appreciated if you split this next time, if not in separate PRs, at least in separate commits.

Reply:
Sorry for that! I was trying to update golang version, get this into the official Fedora repository at the same time, and this is the only working replacement for that dependency. 🤕

Reply:
Writing this from my Fedora: thanks! :pray: :grin: 

If you need any other change in the repo to make your life easier, do not hesitate to reach out or send PRs :wink: